### PR TITLE
Update SRG-APP-000092-CTR-000165 for OCP4 STIG

### PR DESCRIPTION
**Pull request by DhritiShikhar**
#### Description:

- automated check present -> added the rule
- added check and fixtext

#### Rationale:

- OCP4 STIG content

### CODE REVIEW

File: controls/srg_ctr/SRG-APP-000092-CTR-000165.yml [hunk 1]
```diff
@@ -3,4 +3,165 @@ controls:
   levels:
   - medium
   title: {{{ full_name }}} must initiate session auditing upon startup.
-  status: pending
+  rules:
+  - cluster_logging_operator_exists
```
Comment (jhrozek) [inline controls/srg_ctr/SRG-APP-000092-CTR-000165.yml:7]: typo: the rule is called `cluster_logging_operator_exist` (no `s`). Please also add the SRG reference to the rule.

File: controls/srg_ctr/SRG-APP-000092-CTR-000165.yml [hunk 2]
```diff
@@ -3,4 +3,165 @@ controls:
...
+  - audit_log_forwarding_enables
```
Comment (jhrozek) [inline controls/srg_ctr/SRG-APP-000092-CTR-000165.yml:8]: This rule doesn't contain an SRG reference to this control. Can you also check, while you are touching that rule, if the other two SRG references make sense for the rule?

**Review by jhrozek — CHANGES_REQUESTED** [review]

File: controls/srg_ctr/SRG-APP-000092-CTR-000165.yml [hunk 3]
```diff
@@ -3,4 +3,165 @@ controls:
...
+          - rsyslog-infra
+    ...  
+    EOF
+
+    Please note that many log forwarding destinations are
+    supported, and the fix does not require that you forward your
+    audit logs to rsyslog over mTLS. To better understand how to
+    configure the ClusterLogForwarder for your infrastructure and
+    environment, please consult the OpenShift Logging documentation:
+    https://docs.openshift.com/container-platform/4.8/logging/cluster-logging-external.html
```
Comment (jhrozek) [inline controls/srg_ctr/SRG-APP-000092-CTR-000165.yml:167]: Let's also use unversioned doc link

**Comment by jan-cerny** [comment]
@DhritiShikhar What are your plans?

**Comment by jhrozek** [comment]
> @DhritiShikhar What are your plans?

Dhriti no longer works at RH and I had a long vacation. Sorry about the delay, I'm working through the PRs now.

**Comment by jhrozek** [comment]
The latest push also includes content from PR #8970 so that we don't have a conflict between the two.

**Review by jhrozek — APPROVED** [review]
I adressed my own concerns, so I'm gong to push this PR...

**Pull request merged**